### DISCUSSION TITLE
Unfortunately, we weren't able to approve your educational discount request for @flyingfish7777.

### DISCUSSION THREAD

**Discussion by flyingfish7777** [community/community#52236] [GitHub Education]
I have applied 13 times, but my application has not been approved yet.

**Comment by samus-aran** [comment]
Hi @flyingfish7777 👋 

We'd love to help! You are more likely to get a useful response if you are explicit about the feedback given to you from the rejected application submission. Do you have a screenshot of the rejection reasons? This might help someone give you a nudge in the right direction.
